website/src/router.mjs

```diff
@@ -157,7 +157,12 @@ export class Router {
   async renderEJSFile(ejsFilePath, { currentURI }) {
     let childProcess = await renderEJSChildProcessPool.takeAsync();
     try {
-      return await childProcess.renderAsync({ currentURI, ejsFilePath });
+      let page = await childProcess.renderAsync({ currentURI, ejsFilePath });
+      let postProcess = page.replaceAll(/\r?\n|\r/g, "");
```

Comment:
Must fix: This line breaks many code blocks. See http://127.0.0.1:9001/errors/E0007/ for example.